docs/traversal.md

```diff
@@ -22,11 +22,11 @@ csstree.walk(ast, function(node) {
 
 ## walk(ast, handler)
 
-Visits each node of AST in natural way and calls one or two handlers for each one. 
+Visits each node of AST in a natural way and calls a handler for each one. 
```

Comment:
Actually, this sentence probably needs to be completely reworked. The truth is that you visit each node in a natural way and calls *either* a function or multiple functions mentioned in an object.

Reply:
Suggestions are welcome ;)